bson/_cbsonmodule.c

```diff
@@ -455,49 +455,6 @@ static int write_string(buffer_t buffer, PyObject* py_string) {
     return 1;
 }
 
-/*
- * Are we in the main interpreter or a sub-interpreter?
- * Useful for deciding if we can use cached pure python
- * types in mod_wsgi.
```

Comment:
This pattern is no longer needed because multi-phase init solves removes the shared module state. Each interpreter gets it's own modules state and reloads all the objects now.